core/session.ts

```diff
@@ -293,6 +317,18 @@ export class Session {
     return newTest;
   }
 
+  public declare(fullyQualifiedDataset: { schema: string; name: string }): Declaration {
```

Comment:
You'll notice that in my yet-to-be-approved change this structure `{schema: string, name: string}` has become an interface in Session.ts which I paste here in case you want  to use it too:

export interface IResolvable {
  schema: string;
  name: string;
}

Reply:
Thanks, that's a good idea. I'm going to hold off for now - I left a comment suggesting that we can actually make `Resolvable` be either this interface or a string, in which case it wouldn't be good to use here - but if we don't end up doing that, cleaning this up will be pretty simple in any case!